kusto/kusto.go

```diff
@@ -335,7 +326,7 @@ func (c *Client) Mgmt(ctx context.Context, db string, query Stmt, options ...Mgm
 	return iter, nil
 }
 
-func (c *Client) setQueryOptions(ctx context.Context, op errors.Op, query Stmt, options ...QueryOption) (*queryOptions, error) {
+func setQueryOptions(ctx context.Context, op errors.Op, query Stmt, options ...QueryOption) (*queryOptions, error) {
```

Comment:
Isn't this a breaking change? 
(usage of the function changes from instance.Invoke to modul.Invoke)

Reply:
It's a private method (lowercase start), so we can change it at will.